lib/datadog/appsec/contrib/graphql/appsec_trace.rb

```diff
@@ -27,20 +27,10 @@ def execute_multiplex(multiplex:)
             end
 
             multiplex_return
-          end
-
-          private
-
-          def active_trace
-            return unless defined?(Datadog::Tracing)
-
-            Datadog::Tracing.active_trace
-          end
-
-          def active_span
-            return unless defined?(Datadog::Tracing)
+          rescue StandardError => e
```

Comment:
you are actually right, this will also hide exceptions that could be raised in the code of the customer. I am thinking to fully remove this, and add a task to address this [TODO comment](https://github.com/DataDog/dd-trace-rb/blob/06aa9aceb804a42435f2ff4ad9970f55688713e4/lib/datadog/appsec/instrumentation/gateway.rb#L40).

Reply:
@TonyCTHsu what do you think, is it ok to leave this exception handling out of this PR?